release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -22,7 +22,7 @@ compliance, and governance requirements.
 
 Red Hat {product-title}
 (link:https://access.redhat.com/errata/RHBA-2020:1234[RHBA-2020:1234]) is now
-available. This release uses link:https://v1-18.docs.kubernetes.io/docs/setup/release/notes/[Kubernetes 1.18] with CRI-O runtime. New features, changes, and known issues that pertain to
+available. This release uses link:https://kubernetes.io/docs/setup/release/notes/#v1-19-0[Kubernetes 1.19] with CRI-O runtime. New features, changes, and known issues that pertain to
```

Comment:
So, I had a PR (#24322) prepped to make this link update, but was waiting on the URL to work for 1.19. However once 1.19 came out, it doesn't seem like they posted one like this.

My concerns with using this current URL (without the preceding v1-19) is that these will probably break once 1.20 comes out. Not sure if I have an alternative solution to suggest, but bringing up for thoughts, wdyt?

Same for the link on line 127.

Reply:
I see. I'll revert that link on line 25 back to 1.18 for now and rely on your PR to update it when the Kube 1.19 URL is available. I'll just call out Kube 1.19 without the link on line 127 :+1: